Cell: add helper functions for cssClass (#327)
BSI-Business-Systems-Integration-AG/org.eclipse.scout.docs: docs/modules/releasenotes/partials/release-notes-26.1.adoc

// WARNING: {NotReleasedWarning}
//
// * <<New Feature (since 26.1.xyz)>>

[[styles-css-class]]

> == Utility to modify css classes (Scout JS)

Reviewer: Convention is (or at least was) to use title case for titles. See comment on top of file:

> Use "title case" in chapter titles (https://english.stackexchange.com/questions/14/)

The `styles` utility was extended with new methods to modify the `cssClass`-string of objects or check whether a css class is set or not.
These methods are:

> * `addCssClass`  adds one or more css classes

Reviewer: ```suggestion
* `addCssClass` adds one or more css classes
```
(or maybe add a `:` after each method name to create a name-definition list)

* `removeCssClass` removes one or more css classes
* `toggleCssClass` toggles, i.e. adds or removes, one or more css classes depending on a condition
* `hasCssClass` checks whether one or more css classes are set
* `cssClassAsArray` splits a `cssClass`-string into an array of css classes

> In addition, some new methods have been added to easier modify the `cssClass` property of a `Cell` or check whether a css class is set or not: `addCssClass`, `removeCssClass`, `toggleCssClass`, `hasCssClass`.

Reviewer: ```suggestion
In addition, similar methods have been added to the `Cell` class to easier query and modify the `cssClass` property: `addCssClass`, `removeCssClass`, `toggleCssClass`, `hasCssClass`.
```
